packages/browser-sdk/src/client.ts

```diff
@@ -0,0 +1,260 @@
+import { API_HOST, SSE_REALTIME_HOST } from "./config";
+import {
+  Company,
+  feedback,
+  Feedback,
+  FeedbackOptions as FeedbackOptions,
+  LiveSatisfaction,
+  RequestFeedbackOptions,
+  TrackedEvent,
+  User,
+} from "./feedback/feedback";
+import { FlagsOptions, FlagsClient } from "./flags/flags";
+import { Logger, loggerWithPrefix, quietConsoleLogger } from "./logger";
+import { HttpClient } from "./httpClient";
+import * as feedbackLib from "./feedback/ui";
+import { BucketContext } from "./context";
+
+const isMobile = typeof window !== "undefined" && window.innerWidth < 768;
```

Comment:
Is this the best way to detect that?

Reply:
Probably not, but it's from the tracking-sdk so keeping it as-is for now.